src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/users.service');
+
+const getAllExpenses = (req, res) => {
+  const expenses = expensesService.getAll(req.query);
+
+  res.json(expenses);
+};
+
+const getOneExpense = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expensesService.getById(+id);
+
+  if (!expense) {
+    res.statusCode = 404;
+    res.statusMessage = 'Error';
+    res.end();
+  }
+  res.json(expense);
+};
+
+const createExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const user = userService.getById(userId);
+
+  if (!user || !spentAt || !title || !amount || !category || !note || !userId) {
```

Comment:
It's better to move the user request validation check to the middleware
(Create the middleware folder with needed validation middleware)